packages/client/src/runtime/core/types/exported/README.md

```diff
@@ -0,0 +1,9 @@
+# Exported types
+
+When types from `@prisma/runtime` are used in third party type declaration (for example, extensions that are published to `npm`) we need to ensure that whole type tree is exported from the runtime too. Otherwise, non-exported type can be inlined at each instance they are used in declaration file. That in turn will either affect typechecking performance or simply crash TS compiler if inlined type is too big.
```

Comment:
What is `@prisma/runtime`? it looks like a new npm package to me and I'm confused about that.

Reply:
```suggestion
When types from `@prisma/client/runtime` are used in third party type declaration (for example, extensions that are published to `npm`) we need to ensure that whole type tree is exported from the runtime too. Otherwise, non-exported type can be inlined at each instance they are used in declaration file. That in turn will either affect typechecking performance or simply crash TS compiler if inlined type is too big.
```